bundles/org.openhab.binding.solarman/README.md

```diff
@@ -143,6 +144,18 @@ This is the list you get for the `deye_sg04lp3` inverter type:
 | upload-total-load-consumption            | Number | R            | Total Load Consumption \[0x020F,0x0210\]              |
 | upload-total-load-power                  | Number | R            | Total Load Power \[0x028D\]                           |
 
+## Writable Channels
+
+If you want to write into registers of invertor you need to directly define your channels as writable as by default channels are counted as read only. For example, `deye_sg01hp3` has full set of writable channels connected to Time Of Use functionality, but if you want to add custom writable channels you need to define it like this:
```

Comment:
of the inverter
read only -> read-only
has full set -> has a full set
`Time Of Use`

Reply:
Fixed